src/processdata.c

```diff
@@ -1918,13 +1916,13 @@ int DumpStateH5(Link* sys, unsigned int N, int* assignments, GlobalVars* globals
     }
     //Sending data to proc 0
     else
-    {
+    {	
         //Check for error while opening the file
         MPI_Bcast(&res, 1, MPI_UNSIGNED, 0, MPI_COMM_WORLD);
         if (res)
             return 1;
 
-        unsigned int dim = 4;
+        unsigned int dim = 4;  // adlzanchetta: I have no idea why this was hardcoded
```

Comment:
I think I know why it's hardcoded... It is related to DA where links have a variable number of states (extra states for sensitivity equations). But we only need to save the 4 first ones for 252, the model I used for testing. I should have reversed this changes and find a better solution.

Reply:
Thank you very much for the explanation. I commented and added a note for future generations.
The commit failed on Travis CI. I will probably just accept it this time as it is compiling properly at local and do the appropriate steps on the next release, that is close to come.
By the ways, what is your recommendation on that? Set up Travis for my local repo clone an ensure its compile there before Pull Request new releases?